lms/envs/common.py

```diff
@@ -1003,6 +1003,7 @@
 ZENDESK_URL = None
 ZENDESK_USER = None
 ZENDESK_API_KEY = None
+ZENDESK_CUSTOM_FIELDS = {} # Maps custom field name to Zendesk ID. For example, { 'a_custom_field': 123456 }
```

Comment:
Will this provide a default value for this field in all environments, even if the environment lacks a specific config?